<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLoadSymbol</class>
 <widget class="QDialog" name="DialogLoadSymbol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>418</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>基本符号</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>5</y>
     <width>261</width>
     <height>361</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_symbol">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="BtnFirstPage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;&lt;-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnPrePage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnNextPage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>-&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnLastPage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>-&gt;&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_Btn">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>370</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>60</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="stackedWidget_BtnPage1">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>11</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>样式:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Cb_SymbolGroup_2">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>7</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnReLoad">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>6</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>更新符号库</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="BtnOK">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确 定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnCancel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>取 消</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>31</y>
     <width>321</width>
     <height>381</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>6</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>索引:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>5</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
